examples/linear_model/plot_bayesian_ridge.py

```diff
@@ -15,6 +15,11 @@
 
 The estimation of the model is done by iteratively maximizing the
 marginal log-likelihood of the observations.
+
+We also plot predictions and uncertainties for polynomial Bayesian Ridge.
```

Comment:
I find the terms "polynomial Bayesian Ridge" etc a bit awkward: the estimator isn't polynomial, the problem is.